.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -349,11 +333,10 @@ jobs:
         uses: cilium/cilium/.github/actions/conn-disrupt-test@v1.13
         with:
           job-name: ipsec-downgrade-${{ matrix.name }}
-          # Disable no-missed-tail-calls due to https://github.com/cilium/cilium/issues/26739
-          extra-connectivity-test-flags: --test '!no-missed-tail-calls'
+          # Disable no-unexpected-packet-drops (formerly no-missed-tail-calls)
+          # due to https://github.com/cilium/cilium/issues/26739
+          extra-connectivity-test-flags: --test '!no-unexpected-packet-drops'
```

Comment:
That doesn't look correct actually. We were skipping only Missed tail calls and now we'll skip all packet drops. We should use `--expected-drop-reasons` instead. No sure where my head was at.

Reply:
@pchaigno Ouch, yeah I see what you mean. My bad. Are you working on a fix? If not, I can do it today (and for https://github.com/cilium/cilium/pull/30211).

Reply:
I'm not. Go for it :slightly_smiling_face:

Reply:
Nop, `no-missed-tail-calls` doesn't exist anymore (unless I messed up).

Reply:
yeah I didn't ask the right question. I meant if I should also add missed tail calls as an expected drop reason? I assume we still need to skip those failures.

Reply:
(I suppose we need `--expected-drop-reasons=+Missed tail call`?)

Reply:
Ah, yes. We need both Missed tail calls and CT buffer drops to be allowlisted.

Reply:
I see Robin has this covered in https://github.com/cilium/cilium/pull/30249